gitgud/operations.py

```diff
@@ -21,6 +22,7 @@ def __init__(self, path, initialize_repo=True):
             self.repo = None
 
         self.git_path = os.path.join(self.path, '.git')
+        self.commits_json_path = os.path.join(self.path, '.git', 'gud', 'commits.json')
```

Comment:
Isn't this out of scope for this PR?

Reply:
It's there because all features (abort_commit, track_merges, track_rebases) are based off the track_commits branch.